docs/source/json-api/index.rst

```diff
@@ -431,7 +431,7 @@ HTTP Response
         "status": 200,
         "result": {
             "exerciseResult": "#201:1",
-            "contracts": [
+            "events": [
```

Comment:
you did not update the Where section that explains what `contracts` now `events` element is

Reply:
a9afe39b5e